travis/test_pylint

```diff
@@ -1,9 +1,47 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-"""
-Script to process pylint run and exit with result.
-"""
+
+'''This script runs `pylint` with plugins and special configuration files.
```

Comment:
I did not find this guideline here: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
Then, you can create a PR to MT with this rule.